.github/workflows/deployment.yml

```diff
@@ -24,16 +24,16 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
With the new localization on next not being supported with node 20 will updating deploy to package and publish with 20 be an issue?
@t1m0thyj @rudyflores FYI

Reply:
We should be fine because we're still setting up Node 18 to use in the workflow. The only thing changing to Node 20 is the version of Node that 3rd-party GitHub Actions use: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/